### PR TITLE
docs: moving best examples into subdirs

### PR DESCRIPTION
**Pull request by staceysv**
* added app, reference, tutorial under examples
* moves some of the best examples into these

### CODE REVIEW

**Comment by scottire** [comment]
- Can we move `app` to `experimental`, I'll let Tristan know
- Can we move `control` `layout` `panel/plot` and `viz` under `reference`.
- Can we rename `tutorial` to `tutorials`
- `apps` is very empty now and we don't have immediate plans to improve that
-- Is there anything that can go in there now from `experimental`/`app`?
-- If not, can we move the quickstart in `apps` to `tutorial` and delete `apps` for now.
--- In the future, this will be where we can keep instructions to use different templates

**Comment by staceysv** [comment]
- yes, let's put app/ under experimental & all the others under reference—thanks for doing that! eventually may want to re-org and flatten.
- renamed tutorial => tutorials, removed all the other empty folders
- love what you did with getting started, numbers, and apps! i personally don't like the "app" / "apps" name because it doesn't mean anything to me—let's go with this structure and see what else we can improve/add.

**Comment by scottire** [comment]
> i personally don't like the "app" / "apps" name because it doesn't mean anything to me—let's go with this structure and see what else we can improve/add
agreed, I was torn between that and `gallery`. Went with gallery instead

**Review by scottire — APPROVED** [review]

File: integration_test/cypress/e2e/testlib.ts
Comment (staceysv) [inline integration_test/cypress/e2e/testlib.ts:1]: @dannygoldstein could you double-check this? minor change but could affect tests

File: weave-js/src/components/Panel2/PanelBoolean/Component.tsx
Comment (staceysv) [inline weave-js/src/components/Panel2/PanelBoolean/Component.tsx:1]: @dannygoldstein also a quick check here—just adding an id to this component so we can use a PanelBoolean in notebooks and pass notebook tests

**Comment by staceysv** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by jlzhao27 — APPROVED** [review]

**Pull request merged**